packages/expo-splash-screen/README.md

```diff
@@ -0,0 +1,444 @@
+# expo-splash-screen
+
+`expo-splash-screen` customizes the splash screen that is shown on application launch. A splash screen is what users see when the app is launched, before it has loaded. Splash screens (sometimes called launch screens) provide a user's first experience with your application.
+
+## Content
+
+- [📜	CHANGELOG](./CHANGELOG.md)
+- [🚀 Features](#-features)
+- [📚 API](#-api)
+- [🗒 Examples](#-examples)
+- [💻 Installation in managed Expo projects](#-installation-in-managed-expo-projects)
+- [🖥 Installation in bare React Native projects](#-installation-in-bare-react-native-projects)
+  - [📱 Configure iOS](#-configure-ios)
+  - [🤖 Configure Android](#-configure-android)
+- [👏 Contributing](#-contributing)
+- [❓ Known issues](#-known-issues)
+- [🏅 Hall of fame](#-hall-of-fame)
+
+## 🚀 Features
+
+### Built-in splash screen image resize modes
+
+`expo-splash-screen` is shipped with built-in feature for taking care of proper displaying your splash screen image. You can use following the resize modes to obtain behavior as if using React Native `<Image>` component's `resizeMode` style.
+
+#### `CONTAIN` resize mode
+
+Scale the image uniformly (maintain the image's aspect ratio) so that both dimensions (width and height) of the image will be equal to or less than the corresponding dimension of the device's screen.
+
+| Android                                                   | iOS                                                        |
+|-----------------------------------------------------------|------------------------------------------------------------|
+| <img src="./assets/demo-android-contain.gif" height="350" /> | <img src="./assets/demo-ios-contain.gif" height="350" />  |
+
+#### `COVER` resize mode
+
+Scale the image uniformly (maintain the image's aspect ratio) so that both dimensions (width and height) of the image will be equal to or larger than the corresponding dimension of the device's screen.
```

Comment:
```suggestion
Scale the image uniformly (maintaining the image's aspect ratio) so that both the width and height of the image will be equal to or larger than the corresponding dimension of the device's screen.
```